net/base/cache_type.h

```diff
@@ -24,9 +24,6 @@ enum BackendType {
   CACHE_BACKEND_DEFAULT,
   CACHE_BACKEND_BLOCKFILE,  // The |BackendImpl|.
   CACHE_BACKEND_SIMPLE,     // The |SimpleBackendImpl|.
-#if defined(STARBOARD)
```

Comment:
Im a bit confused on whether to us this or USE_COBALT_CUSTOMIZATIONS

Reply:
This line is being removed, so probably doesn't matter